src/api/routes/authRouter.js

```diff
@@ -22,9 +23,15 @@ const {
   login
 } = authController;
 const { usernameExists, emailExists } = userValidation;
-const { verifyToken } = Auth;
+const { verifyToken, checkIsAdmin } = Auth;
 
 const { google, twitter } = socialAccount;
+const { getTermsAndConditions, updateTermsAndConditions } = termsAndConditions;
+
+// terms and conditions
+
+authRouter.get('/termsandconditions/:id', getTermsAndConditions);
```

Comment:
Should update Terms and conditions be in auth part? What do you think? I think it is a little bit different. It should have a different router.

Reply:
What do you suggest? should I create a new file? I thought it was okay since we were updating terms and conditions used by a user who wants to sign up but I can create a new file for those two endpoints